src/viewProviders/schemas.ts

```diff
@@ -119,8 +138,13 @@ export class SchemasViewProvider implements vscode.TreeDataProvider<SchemasViewP
 
   getParent(element: SchemasViewProviderData): SchemasViewProviderData | null {
     if (element instanceof Schema) {
-      // if we're a schema, our parent is our Subject.
-      return element.subjectObject();
+      // if we're a schema, our parent is the schema's Subject as found in our map
+      const subject = this.subjectsInTreeView.get(element.subject);
+      if (!subject) {
+        logger.error("Strange, couldn't find subject in tree view", { element });
+        return null;
```

Comment:
Does this path show up in the UI as that generic VS Code `Cannot resolve tree item for element ...` error notification?

Reply:
No. I don't think I was able to hit this statement, but want a trail if able to in the future.